src/main/kotlin/net/atos/zac/zoeken/IndexingService.kt

```diff
@@ -184,23 +184,16 @@ class IndexingService @Inject constructor(
             rows = SOLR_MAX_RESULTS
         }
         var cursorMark = CursorMarkParams.CURSOR_MARK_START
-        var done = false
-        while (!done) {
+        while (true) {
```

Comment:
No alternative for a `while(true)` looop?

Reply:
Tried to find a better one, but pagination makes it hard to get rid of while. 
Solr documentation also uses while (!done) which is even more verbose

Reply:
ok, no problem!